test/create-busybox-container

```diff
@@ -9,6 +9,16 @@ mkdir -p "$root"
 mkdir "$root/bin"
 cp $(type -P busybox) "$root/bin"
 
+ID_LIKE=$(awk -F= '$1=="ID_LIKE" { print $2 ;}' /etc/os-release)
+if [[ "$ID_LIKE" = *"suse"* ]]; then
+    mkdir -p "$root/lib"
+    mkdir -p "$root/lib64"
```

Comment:
Does suse really not do usr-merge?

Reply:
https://en.opensuse.org/openSUSE:Usr_merge says that opensuse does...

Reply:
> Does suse really not do usr-merge?

That is work in progress but still not finished.
Some shared libraries, needed in the container, still reside below /lib for 32bit and /lib64 for 64bit.

Reply:
I think it's one of those case where "ripping off the band-aid" is the way to go... A brief moment of pain is better than dragging this out and dealing with workarounds for workarounds.